### ISSUE TITLE
Bug: Multiple eslint-disable/enable comments on same line do not work

### ISSUE THREAD

**Issue by Splines** [eslint/eslint#17779]
### Environment

Node version: v18.17.1
npm version: v9.6.7
Local ESLint version: v8.53.0
Global ESLint version: Not found
Operating System: linux 5.15.133.1-microsoft-standard-WSL2

### What parser are you using?

Default (Espree)

### What did you do?

Try to lint this line with ESLint:
```js
console.log(/* eslint-disable */X/* eslint-enable */ /* eslint-disable */X/* eslint-enable */   );
```

### What did you expect to happen?

Expected that no error is shown on second `X` token.

### What actually happened?

We get a `Parsing error: Unexpected token X` for the second `X` token.

### Link to Minimal Reproducible Example

[Link](https://eslint.org/play/#eyJ0ZXh0IjoiY29uc29sZS5sb2coLyogZXNsaW50LWRpc2FibGUgKi9YLyogZXNsaW50LWVuYWJsZSAqLyAvKiBlc2xpbnQtZGlzYWJsZSAqL1gvKiBlc2xpbnQtZW5hYmxlICovICAgKTsiLCJvcHRpb25zIjp7ImVudiI6eyJlczYiOnRydWV9LCJwYXJzZXJPcHRpb25zIjp7ImVjbWFGZWF0dXJlcyI6e30sImVjbWFWZXJzaW9uIjoibGF0ZXN0Iiwic291cmNlVHlwZSI6InNjcmlwdCJ9LCJydWxlcyI6eyJjb25zdHJ1Y3Rvci1zdXBlciI6WyJlcnJvciJdLCJmb3ItZGlyZWN0aW9uIjpbImVycm9yIl0sImdldHRlci1yZXR1cm4iOlsiZXJyb3IiXSwibm8tYXN5bmMtcHJvbWlzZS1leGVjdXRvciI6WyJlcnJvciJdLCJuby1jYXNlLWRlY2xhcmF0aW9ucyI6WyJlcnJvciJdLCJuby1jbGFzcy1hc3NpZ24iOlsiZXJyb3IiXSwibm8tY29tcGFyZS1uZWctemVybyI6WyJlcnJvciJdLCJuby1jb25kLWFzc2lnbiI6WyJlcnJvciJdLCJuby1jb25zdC1hc3NpZ24iOlsiZXJyb3IiXSwibm8tY29uc3RhbnQtY29uZGl0aW9uIjpbImVycm9yIl0sIm5vLWNvbnRyb2wtcmVnZXgiOlsiZXJyb3IiXSwibm8tZGVidWdnZXIiOlsiZXJyb3IiXSwibm8tZGVsZXRlLXZhciI6WyJlcnJvciJdLCJuby1kdXBlLWFyZ3MiOlsiZXJyb3IiXSwibm8tZHVwZS1jbGFzcy1tZW1iZXJzIjpbImVycm9yIl0sIm5vLWR1cGUtZWxzZS1pZiI6WyJlcnJvciJdLCJuby1kdXBlLWtleXMiOlsiZXJyb3IiXSwibm8tZHVwbGljYXRlLWNhc2UiOlsiZXJyb3IiXSwibm8tZW1wdHkiOlsiZXJyb3IiXSwibm8tZW1wdHktY2hhcmFjdGVyLWNsYXNzIjpbImVycm9yIl0sIm5vLWVtcHR5LXBhdHRlcm4iOlsiZXJyb3IiXSwibm8tZXgtYXNzaWduIjpbImVycm9yIl0sIm5vLWV4dHJhLWJvb2xlYW4tY2FzdCI6WyJlcnJvciJdLCJuby1leHRyYS1zZW1pIjpbImVycm9yIl0sIm5vLWZhbGx0aHJvdWdoIjpbImVycm9yIl0sIm5vLWZ1bmMtYXNzaWduIjpbImVycm9yIl0sIm5vLWdsb2JhbC1hc3NpZ24iOlsiZXJyb3IiXSwibm8taW1wb3J0LWFzc2lnbiI6WyJlcnJvciJdLCJuby1pbm5lci1kZWNsYXJhdGlvbnMiOlsiZXJyb3IiXSwibm8taW52YWxpZC1yZWdleHAiOlsiZXJyb3IiXSwibm8taXJyZWd1bGFyLXdoaXRlc3BhY2UiOlsiZXJyb3IiXSwibm8tbG9zcy1vZi1wcmVjaXNpb24iOlsiZXJyb3IiXSwibm8tbWlzbGVhZGluZy1jaGFyYWN0ZXItY2xhc3MiOlsiZXJyb3IiXSwibm8tbWl4ZWQtc3BhY2VzLWFuZC10YWJzIjpbImVycm9yIl0sIm5vLW5ldy1zeW1ib2wiOlsiZXJyb3IiXSwibm8tbm9ub2N0YWwtZGVjaW1hbC1lc2NhcGUiOlsiZXJyb3IiXSwibm8tb2JqLWNhbGxzIjpbImVycm9yIl0sIm5vLW9jdGFsIjpbImVycm9yIl0sIm5vLXByb3RvdHlwZS1idWlsdGlucyI6WyJlcnJvciJdLCJuby1yZWRlY2xhcmUiOlsiZXJyb3IiXSwibm8tcmVnZXgtc3BhY2VzIjpbImVycm9yIl0sIm5vLXNlbGYtYXNzaWduIjpbImVycm9yIl0sIm5vLXNldHRlci1yZXR1cm4iOlsiZXJyb3IiXSwibm8tc2hhZG93LXJlc3RyaWN0ZWQtbmFtZXMiOlsiZXJyb3IiXSwibm8tc3BhcnNlLWFycmF5cyI6WyJlcnJvciJdLCJuby10aGlzLWJlZm9yZS1zdXBlciI6WyJlcnJvciJdLCJuby11bmRlZiI6WyJlcnJvciJdLCJuby11bmV4cGVjdGVkLW11bHRpbGluZSI6WyJlcnJvciJdLCJuby11bnJlYWNoYWJsZSI6WyJlcnJvciJdLCJuby11bnNhZmUtZmluYWxseSI6WyJlcnJvciJdLCJuby11bnNhZmUtbmVnYXRpb24iOlsiZXJyb3IiXSwibm8tdW5zYWZlLW9wdGlvbmFsLWNoYWluaW5nIjpbImVycm9yIl0sIm5vLXVudXNlZC1sYWJlbHMiOlsiZXJyb3IiXSwibm8tdW51c2VkLXZhcnMiOlsiZXJyb3IiXSwibm8tdXNlbGVzcy1iYWNrcmVmZXJlbmNlIjpbImVycm9yIl0sIm5vLXVzZWxlc3MtY2F0Y2giOlsiZXJyb3IiXSwibm8tdXNlbGVzcy1lc2NhcGUiOlsiZXJyb3IiXSwibm8td2l0aCI6WyJlcnJvciJdLCJyZXF1aXJlLXlpZWxkIjpbImVycm9yIl0sInVzZS1pc25hbiI6WyJlcnJvciJdLCJ2YWxpZC10eXBlb2YiOlsiZXJyb3IiXX19fQ==)

### Participation

- [ ] I am willing to submit a pull request for this issue.

Maybe related [code](https://github.com/eslint/eslint/blob/4a88a54aeb456abe94da9f5b3d5472d14b3116f1/lib/linter/linter.js#L397-L409).

### Additional comments

Admittedly, this statement is probably not what one would encounter in real-world examples. However, I'd like to rely on this to work for a custom processor which replaces unknown tokens by such a block:`/* eslint-disable */X/* eslint-enable */`.

**Comment by Rec0iL99** [comment]
Hi @Splines, thanks for the issue. The example produces a parsing error because the arguments to `console.log()` are not separated by a `,`.

console.log(X, X);

[playground](https://eslint.org/play/#eyJ0ZXh0IjoiY29uc29sZS5sb2coLyogZXNsaW50LWRpc2FibGUgKi9YLyogZXNsaW50LWVuYWJsZSAqLywgLyogZXNsaW50LWRpc2FibGUgKi9YLyogZXNsaW50LWVuYWJsZSAqLyAgICk7Iiwib3B0aW9ucyI6eyJlbnYiOnsiZXM2Ijp0cnVlfSwicnVsZXMiOnt9LCJwYXJzZXJPcHRpb25zIjp7ImVjbWFGZWF0dXJlcyI6e30sImVjbWFWZXJzaW9uIjoibGF0ZXN0Iiwic291cmNlVHlwZSI6InNjcmlwdCJ9fX0=)

**Comment by Splines** [comment]
Hey @Rec0iL99, thanks for your reply. With an additional comma it works for sure. However, if

```js
console.log(/* eslint-disable */X/* eslint-enable */);
```

is valid, I'd expect this to also be valid:

```js
console.log(/* eslint-disable */X/* eslint-enable */ /* eslint-disable */X/* eslint-enable */);
```

since from a logical standpoint, one can reduce the second case to the first one.

(Note that adding in the `,` programmatically for the processor I'm writing would heavily increase the overall complexity.)

**Comment by Rec0iL99** [comment]
```js
console.log(/* eslint-disable */X/* eslint-enable */ /* eslint-disable */X/* eslint-enable */);
```

This code is not valid JavaScript (syntax error).

**Comment by Splines** [comment]
Oh gosh, thanks, I think I was too immersed with solving index mapping problems for an [eslint plugin](https://github.com/Splines/eslint-plugin-erb) that I forgot this was not even valid JavaScript 😅 Then you can declare this issue as null and void ;)
